README.md

```diff
@@ -27,7 +27,8 @@ Todo steps??
 | `CHT_COMPOSE_PATH`   | yes      | Absolute path to the folder where the CHT docker-compose file is saved.                                                                                                                                                                      | 
 | `COUCHDB_USER`       | yes      | CouchDb main admin account username.                                                                                                                                                                                                         |
 | `COUCHDB_PASSWORD`   | yes      | CouchDb main admin account password.                                                                                                                                                                                                         | 
-| `COUCHDB_SECRET`     | yes      | Secret that is used by the CouchDB peers to communicate with each other and to generate authentication cookies. This field is mandatory if you are running CouchDb clustered mode to make the same authentication cookie valid on all nodes. |  
+| `COUCHDB_SECRET`     | yes      | Secret that is used by the CouchDB peers to communicate with each other and to generate authentication cookies. This field is mandatory if you are running CouchDb clustered mode to make the same authentication cookie valid on all nodes. |
+| `COUCHDB_DATA`       | yes      | Absolute path to the folder that will serve as CouchDb data location.                                                                                                                                                                        |
```

Comment:
I we should have an optional COUCHDB_CONFIG variable too. Not a blocker for this change.

Reply:
I don't believe we actually use that now, what were you thinking it should provide?

So sorry for the lack of context for this PR :) I just realized there's just a mouthful to explain

I'm adding this environment variable so it gets passed to the self-hosted couch here: https://github.com/medic/cht-core/blob/archv3-e2e-upgrade/scripts/build/cht-couchdb-single-node.yml.template#L8

```
 volumes:
      - ${COUCHDB_DATA:-./srv}:/opt/couchdb/data
      - cht-credentials:/opt/couchdb/etc/local.d/
```

If the path was missing, there's a really weird interaction: 
- as you can see, the default path is `./srv` 
- the in-container path for the `docker-compose` folder in `cht-upgrade-service` is `/docker-compose`: https://github.com/medic/cht-upgrade-service/blob/main/docker-compose.yml#L10
```
volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ${DOCKER_CONFIG_PATH}:/config
      - ${CHT_COMPOSE_PATH}:/docker-compose
```
- if the `cht-upgrade-service` starts couchdb with the default relative path of `./srv`, the absolute path will be computed as `/docker-compose/srv`, **but** the docker "path resolver" (I don't know what it is exactly so I'm dubbing it) resolves this path at **host level** to `/docker-compose/srv` instead of `<path we passed to cht-upgrade-service that it mounts to /docker-compose>/srv`. 

I'm going to provide a clear example: 

- **host** `CHT_COMPOSE_PATH` passed to `cht-upgrade-service` is `/home/gareth/cht`. 
- `cht-upgrade-service` starts couchdb without a `COUCHDB_DATA` path
- **host** path where couchdb will store data will be `/docker-compose/srv`. it will **not** be `/home/gareth/cht/srv` 

Let me know what you think.

Reply:
Makes sense, and I think making this a required variable is good.